### DISCUSSION TITLE
AR is not working with iOS Chrome or Edge

### DISCUSSION THREAD

**Discussion by hbaugmentedreality** [google/model-viewer#3389] [Q&A]
Hi Guys,
Anyone is having this issue with AR in iOS?
When I press "View in your space" button, the system breaks down.   
(see the video)
Is happens with any model.    But works fine in Safari.

https://user-images.githubusercontent.com/73670108/165226008-c3a192d7-708d-4e2c-bc58-b569b34f3d1b.mp4

Any suggestions? 
iOS version:   15.4.1
Chrome version:  100.0.4896.85

Thanks

**Comment by elalish** [comment]
Yeah, there are some known issues with iOS Chrome they are working on. See #3235 and https://github.com/google/model-viewer/issues/3292#issuecomment-1106736713
